docs/shared/tabs.js

```diff
@@ -79,19 +79,24 @@ function switchAllTabs(language) {
         for (let tabcontent of tabcontents.children) {
             let button = document.createElement("button");
             button.dataset.lang = tabcontent.dataset.lang;
+            button.innerText = tabcontent.dataset.lang;
             button.classList.add("tablinks");
             if (!tabcontent.innerHTML) {
                 button.classList.add("disabled");
-                if (!tabcontent.dataset.bug) {
-                    button.title = `${tabcontent.dataset.lang} does not provide this API`;
+                const tooltip = document.createElement("span");
+                tooltip.classList.add("tooltip");
+                if (tabcontent.dataset.bug) {
+                    tooltip.innerHTML +=
+                    `${tabcontent.dataset.lang} does not provide this API yet. \nFollow <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=${tabcontent.dataset.bug}" target="_blank">Bug ${tabcontent.dataset.bug}</a> for updates.`;
+                } else if (tabcontent.dataset.info) {
+                    tooltip.innerHTML += `${tabcontent.dataset.info}`;
```

Comment:
Ooooooh, these uses of `innerHTML` make my [webdev hackles rise with anxiety](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations). I do see how it's being used to provide rich output, but what if that output executes JS...

If we're going to continue using `innerHTML` please put a comment explaining why this is safe. Or an acceptable risk.

Otherwise I'd really prefer we come up with some other way to provide the richness these tooltips provide. I am really looking forward to being able to click links and select text in these new tooltips.

Reply:
It is possible for me to change this, but then I would need to parse any string passed to `data-info` and actually create the DOM elements which would be _very_ complex. Since the Glean book is a static website I don't see the issue in using `innerHTML` here. From the MDN link:

> there is still a security risk whenever you use innerHTML to set strings over which you have no control.

In this case we do have control as we are the ones defining which HTML string is passed to `innerHTML` and I don't see we ever changing this to actually receive dynamic user input. WDYT?

Reply:
An inline comment to that effect would be very nice and would satisfy my anxiety, yes!